.github/workflows/build-and-test.yml

```diff
@@ -198,7 +198,10 @@ jobs:
 
       - name: Feedz Publish
         shell: pwsh
-        run: |
-          dotnet nuget push .\artifacts\*.nupkg -k ${{secrets.FEEDZ_TOKEN}} -s https://f.feedz.io/sixlabors/sixlabors/nuget/index.json
-          dotnet nuget push .\artifacts\*.snupkg -k ${{secrets.FEEDZ_TOKEN}} -s https://f.feedz.io/sixlabors/sixlabors/symbols
-        # TODO: If github.ref starts with 'refs/tags' then it was tag push and we can optionally push out package to nuget.org
+        run: dotnet nuget push .\artifacts\*.nupkg -k ${{secrets.FEEDZ_TOKEN}} -s https://f.feedz.io/sixlabors/sixlabors/nuget/index.json -ss https://f.feedz.io/sixlabors/sixlabors/symbols
```

Comment:
I'm pretty sure this will publish twice now once for the push on `main` and one for the tag build. I'm not sure how `MinVer` resolves the version on the push on main build, but theres a chance your pipeline could fail because either the CI or the tag build has already published to Feedz and you are trying a second time with the same version.

Reply:
MinVer is great and will actually produce 2 different tags.

1. A minor for main
2. The tag value for the tag build.